server/src/lexer.rs

```diff
@@ -819,7 +819,207 @@ pub fn source_lexer(
                 }
 
                 RegexDelimType::BlockComment(closing_regex) => {
-                    panic!("Unimplemented.")
+                    let full_comment;
+
+                    // A block comment opening delimiter was found. Find the closing delimiter.
+                    // for now just look for star slash
+                    let end_of_comment_index = source_code.find("*/");
```

Comment:
Don't hardcode the delimiter -- each language will have its own. Instead, use something like `closing_regex.find(source_code)` to do this.